### PR TITLE
NEP-17 RPC client

### PR DESCRIPTION
**Pull request by roman-khimov**
Multitransfers need a bit more time since they will be a bit different.

### CODE REVIEW

File: pkg/rpcclient/nep17/nep17.go [hunk 1]
```diff
@@ -0,0 +1,111 @@
+/*
+Package nep17 contains RPC wrappers to work with NEP-17 contracts.
+
+Safe methods are encapsulated into TokenReader structure while Token provides
+various method to perform the only NEP-17 state-changing call, Transfer.
```
[the remark this answers is not in the document]
Comment (roman-khimov) [reply pkg/rpcclient/nep17/nep17.go]: Fixed.

File: pkg/rpcclient/nep17/nep17.go [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+}
+
+// MultiTransfer is not a real NEP-17 method, but rather a convenient way to
+// perform multiple transfers (usually from a single account) in one transaction.
+// It accepts a set of parameters, creates a script that calls `transfer` as
+// many times as needed (with ASSERTs added, so if any of the transfers fail
+// whole transaction (with all transfers) fails). The values returned are the
+// same as in Transfer.
+func (t *Token) MultiTransfer([]TransferParameters) (util.Uint256, uint32, error) {
+	return util.Uint256{}, 0, nil
```
Comment (AnnaShaleva) [inline pkg/rpcclient/nep17/nep17.go:128]: Maybe remove it for now, and add the whole method later?

Comment (AnnaShaleva) [reply pkg/rpcclient/nep17/nep17.go:128]: It wasn't here before, is it added by accident?

Comment (roman-khimov) [reply pkg/rpcclient/nep17/nep17.go:128]: Ouch. Looks like I've committed something that wasn't supposed to be committed at the moment.

Comment (roman-khimov) [reply pkg/rpcclient/nep17/nep17.go]: Fixed.

**Pull request merged**